test/extended/networking/route_advertisements.go

```diff
@@ -488,6 +489,10 @@ var _ = g.Describe("[sig-network][OCPFeatureGate:RouteAdvertisements][Feature:Ro
 					g.By("deploy nmstate handler")
 					err = deployNmstateHandler(oc)
 					o.Expect(err).NotTo(o.HaveOccurred())
+
+					// This is a workaround for OCPBUGS-56488: scale down
+					// nmstate operator and disable liveness probe
+					o.Eventually(workaroundOCPBUGS56488).WithArguments(oc).WithTimeout(3 * time.Minute).WithPolling(5 * time.Second).Should(o.BeTrue())
```

Comment:
Can this end up running in paralel ? Or we have just one test at this BeforeEach ?

Reply:
its only one test, but it wouldn't be an issue if it ran in parallel